telemetry-self-monitor/Dockerfile

```
@@ -0,0 +1,38 @@
+ARG PROMETHEUS_VERSION
+ARG ALPINE_VERSION
+
+FROM node:alpine${ALPINE_VERSION} as builder
+RUN apk add --no-cache git make musl-dev go curl bash
+
+# Configure Go
+ENV GOROOT /usr/lib/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
+
+RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
+WORKDIR /
+RUN curl -LO https://github.com/prometheus/prometheus/archive/refs/tags/v${PROMETHEUS_VERSION}.tar.gz  && \
+    tar xvzf v${PROMETHEUS_VERSION}.tar.gz
+
+WORKDIR /prometheus-${PROMETHEUS_VERSION}
+COPY plugins.yml plugins.yml
+RUN make build
+
+FROM alpine:${ALPINE_VERSION}
+
```

I think these command is important to install latest patches for the alpine libs
```suggestion
RUN apk --no-cache upgrade
```

---

done